ha/const.py

```diff
@@ -151,6 +151,15 @@
 SOURCE_ALERT_FILTER_RULES_FILE = "{}/alert_filter_rules.json".format(SOURCE_CONFIG_PATH)
 SYSTEM_SERVICE_FILE = "{}/event_analyzer.service".format(SYSTEM_DIR)
 SOURCE_SERVICE_FILE = "{}/conf/service/event_analyzer.service".format(SOURCE_PATH)
+ACTUATOR_RESPONSE_TYPE= "actuator_response_type"
```

Comment:
These can also go to AlertConstants Class as these are related to alert?

Reply:
Actually those are common so kept in const file only instead of alert or iem constants.